functions/index.js

```diff
@@ -60,6 +60,57 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.createNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var title = request.body.data.Title;
+        var des = request.body.data.Description;
+        var status = request.body.data.Status;
+        var totalDevelopmentTask = request.body.data.development;
+        var totalBusinessTask = request.body.data.business;
+        var totalMarketingTask = request.body.data.marketing;
+        var lastSprintId = 0;
+        var newtSprintId = lastSprintId + 1;
```

Comment:
What is this, think logically what will happen.... newSprintId will always be 1. Please Think how to solve, I have already given you example to solve this.

Reply:
Okay